enclave-tls/src/include/enclave-tls/tls_wrapper.h

```diff
@@ -45,6 +45,7 @@ typedef struct {
 } tls_wrapper_opts_t;
 
 struct tls_wrapper_ctx {
+	enclave_tls_handle etls_handle;
```

Comment:
Should be `etls_core_context_t *`, and add some comments about why adding this field.

Reply:
enclave_tls_handle belongs to api.h which can be exposed to user.

Reply:
OK.